src/gen/gemini.py

```diff
@@ -0,0 +1,40 @@
+import json
+import google.generativeai as genai
+
+async def call_gemini(prompt="", api_key=None, generation_config=None, safety_settings=None):
+    genai.configure(api_key=api_key)
```

Comment:
Shouldn't this just be configured only once?

Reply:
You are right! Thanks for point this out :)